packages/neon/neon_news/lib/widgets/feed_fab.dart

```diff
@@ -0,0 +1,29 @@
+part of '../neon_news.dart';
+
+class NewsFeedFAB extends StatelessWidget {
```

Comment:
I'm not sure if abbreviating FloatingActionButton is a good idea or not. On the one hand it's a commonly used abbreviation that will be understood by most people, on the other hand we don't abbreviate any other widget name. I would rather spell it out completely, but we can also keep it this way :woman_shrugging: